docs/glossary.md

```diff
@@ -14,27 +14,27 @@ To help you navigate the platform more effectively, familiarize yourself with th
+ |                                                                                                                           |**Deployment Flows** - business entity that describes the whole delivery process of the selected application set via the respective environments. The main idea of the CD pipeline is to promote the application version between the environments by applying the sequential verification (i.e. the second environment will be available if the verification on the first environment is successfully completed). _**NOTE**: The Deployment Flow can include the essential set of applications with its specific environments as well._|
+ |                                                                                                                           |**Deployment Flow Environment** - business entity that is presented as the logical gate required for the application set inspection. Every environment has one OpenShift project where the selected application set is deployed. All the environments are sequential and promote applications one-by-one.|
  |                                                                                                                           |**Codebase** - business entity that possesses a code.|
  |                                                                                                                           |**Codebase Branch** - business entity that represents a specific version in a Git branch. Every codebase branch has a Codebase Docker Stream entity.|
- |                                                                                                                           |**Codebase Docker Stream** - a deployable component that leads to the application build and displays that the last build was verified on the specific stage. Every CD pipeline stage accepts a set of Codebase Image Streams (cbis) that are input and output. _SAMPLE: if an application1 has a master branch, the input cbis will be named as [app name]-[pipeline name]-[stage name]-[master] and the output after the passing of the DEV stage will be as follows: [app name]-[pipeline name]-[stage name]-[dev]-[verified]._|
+ |                                                                                                                           |**Codebase Docker Stream** - a deployable component that leads to the application build and displays that the last build was verified on the specific stage. Every Deployment FLow Environment accepts a set of Codebase Image Streams (cbis) that are input and output. _SAMPLE: if an application1 has a master branch, the input cbis will be named as [app name]-[pipeline name]-[environment name]-[master] and the output after the passing of the DEV environment will be as follows: [app name]-[pipeline name]-[environment name]-[dev]-[verified]._|
```

Comment:
**Codebase Image Stream (CBIS)** - serves as a key component in managing the deployment lifecycle of an application version. It facilitates the identification and progression of application versions that are ready for deployment. Within the Deployment Flow, each environment leverages Codebase Image Streams (CBIS) to manage application versions effectively. For instance, an application named 'application1' associated with a master branch would be initially tagged in the CBIS using the format: `[codebase name]-[branch name]`, e.g., 'application1-main'. As the application version progresses through the Deployment Flow, reaching readiness for deployment in a specific environment, the CBIS naming convention evolves to indicate its deployment flow and target environment, typically following the structure: `[deployment flow name]-[environment name]-[codebase name]`. This approach ensures a clear and systematic tracking of application versions from their initial identification through to their readiness for deployment in various environments within the deployment pipeline.